seed_isort_config.py

```diff
@@ -8,11 +8,16 @@
 import os.path
 import re
 import subprocess
+from collections import OrderedDict
 
 from aspy.refactor_imports.classify import classify_import
 from aspy.refactor_imports.classify import ImportType
 
-
+CONFIG_FILE_TO_START_RE = OrderedDict([
+    ('.isort.cfg', re.compile(r'^(\[settings\])$', re.M)),
+    ('setup.cfg', re.compile(r'^(\[isort\])$', re.M)),
+    ('tox.ini', re.compile(r'^(\[isort\])$', re.M)),
```

Comment:
what if I prefer to have the setting in `tox.ini` over `setup.cfg`? starting to lean towards auto-adding the section on being a bad idea as we can't possibly capture what the user wants (and suggesting a one-time manual operation doesn't feel all that bad)

Reply:
That's a good point. Would you rather have something auto-added but be in the wrong place, or not added at all? I think I would prefer the former as long as I understand that I can move the config elsewhere and everything would still work seamlessly. 

Maybe the logic could be, if `known_third_party` exists in one of the files, or there is an `[isort]` section in `tox.ini`/`setup.cfg`, update it. If not, we could create a new `.isort.cfg`. That is probably the most we can confidently infer (I don't know if adding stuff to people's `.editorconfig` is something that's expected).

Reply:
tbh looking at https://github.com/timothycrosley/isort/blob/d14d122ab03abcd76e3153270cd8b91854baf889/isort/settings.py#L147-L150 makes me think maybe we shouldn't bother